src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -170,7 +184,13 @@ const Actions: React.FC<Props> = ({
     const [driverName, setDriverName] = useState("");
     const [collectionCentre, setCollectionCentre] = useState("");
 
-    const selectedData = Array.from(selected.map((index) => data[index]));
+    const [selectedData, setSelectedData] = useState(
+        Array.from(selected.map((index) => data[index]))
+    );
+
+    React.useEffect(() => {
```

Comment:
It's conventional to import `useEffect` directly like we do `useState` in this file so that we can write this without `React.`